ios/FluentUI/Other Cells/CenteredLabelCell.swift

```diff
@@ -61,18 +102,14 @@ open class CenteredLabelCell: UITableViewCell {
 
     open override func didMoveToWindow() {
         super.didMoveToWindow()
-        if let window = window {
-            label.textColor = Colors.primary(for: window)
-        }
+        updateTokens()
     }
 
     open override func setHighlighted(_ highlighted: Bool, animated: Bool) { }
 
     open override func setSelected(_ selected: Bool, animated: Bool) { }
 
     private struct Constants {
-        static let labelFont: UIFont = Fonts.body
         static let paddingVertical: CGFloat = 11
-        static let defaultHeight: CGFloat = 48
```

Comment:
Looks like you're getting the paddingVertical from the tokens, so you should be able to remove this one.